app/Http/Controllers/BankProviders/GoCardlessController.php

```
@@ -3,60 +3,90 @@
-            'account_id' => $account,
-        ]);
-
         try {
-            $account = Account::findOrFail($account);
+            // Get updateExisting parameter from request, default to true
+            $updateExisting = $request->boolean('update_existing', true);
+            
+            // Get forceMaxDateRange parameter from request, default to false
+            $forceMaxDateRange = $request->boolean('force_max_date_range', false);
+            
+            $result = $this->gocardlessService->syncAccountTransactions($accountId, $request->user(), $updateExisting, $forceMaxDateRange);
```

PHPStan (2.1.15)

Parameter #2 $user of method App\Services\GoCardlessService::syncAccountTransactions() expects App\Models\User, mixed given.


(argument.type)